doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -384,91 +381,104 @@ Rewriting with definitional equality
            exact H.
          Qed.
 
-.. _performingcomputations:
+.. _applyingconversionrules:
 
-Performing computations
----------------------------
+Applying conversion rules
+-------------------------
+
+These tactics apply reductions and expansions, replacing :term:`convertible` terms
+with equivalent terms.
```

Comment:
"Equivalent" may be misinterpreted as referring to setoid equality. What about writing "replacing subterms with definitionally equal subterms"?